databricks/koalas/tests/test_frame_plot.py

```diff
@@ -7,6 +7,7 @@
 import numpy as np
 
 from databricks import koalas
+from databricks.koalas.config import set_option
```

Comment:
@charlesdong1991, how about we just set? Although not all tests are dependent on this configuration but I think it's fine to set globally for the test cases below since this configuration itself is specific to plot and the test cases here are dedicated for plot.

```python
    @classmethod
    def setUpClass(cls):
        super(DataFramePlotTest, cls).setUpClass()
        set_option('plotting.max_rows', 1000)

    @classmethod
    def tearDownClass(cls):
        super(DataFramePlotTest, cls).tearDownClass()
        reset_option('plotting.max_rows')
```

Reply:
nice idea! thanks! @HyukjinKwon